src/component.js

```diff
@@ -34,6 +34,8 @@ export default {
     widgetParams: {
       handler(nextWidgetParams) {
         this.state = null;
+        // TODO: this line here breaks for unmounting a Configure :(
+        // only when `searchClient` is used; not if `apiKey+appId`
```

Comment:
the offending line

Reply:
Do you have a test to reproduce the error? Or a sample?

Reply:
It happens in the storybook, when configure unmounts